src/mono/System.Private.CoreLib/src/System/Reflection/RuntimeParameterInfo.cs

```diff
@@ -256,7 +256,7 @@ int MetadataToken
                 {
                     return GetMetadataToken();
                 }
-                throw new ArgumentException("Can't produce MetadataToken for member of type " + MemberImpl.GetType());
+                throw new ArgumentException(SR.Format(SR.NoMetadataTokenAvailable, MemberImpl.GetType()));
```

Comment:
```suggestion
                throw new ArgumentException(SR.NoMetadataTokenAvailable);
```
See my comment on the message string